android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -410,7 +411,6 @@ class StripeSdkModule(reactContext: ReactApplicationContext, cardFieldManager: S
   @ReactMethod
   fun confirmPaymentMethod(paymentIntentClientSecret: String, params: ReadableMap, options: ReadableMap, promise: Promise) {
     confirmPromise = promise
-    confirmPaymentClientSecret = paymentIntentClientSecret
```

Comment:
@thorsten-stripe  I think this has broken FPX payment since `paymentIntentClientSecret` is used there
https://github.com/stripe/stripe-react-native/blob/f5976da65fbc32947933c6459518b5a67ce3f3b5/android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt#L314